x-pack/plugin/src/test/resources/rest-api-spec/test/transform/preview_transforms.yml

```diff
@@ -150,6 +150,55 @@ setup:
   - match: { generated_dest_index.mappings.properties.by-hour.type: "date" }
   - match: { generated_dest_index.mappings.properties.avg_response.type: "double" }
 
+---
+"Test preview transform latest":
+  - do:
+      transform.preview_transform:
+        body: >
+          {
+            "source": { "index": "airline-data" },
+            "latest": {
+              "unique_key": ["airline"],
+              "sort": "time"
+            }
+          }
+  - match: { preview.0.airline: bar }
+  - match: { preview.0.time: "2017-02-18T01:00:00Z" }
+  - match: { preview.1.airline: foo }
+  - match: { preview.1.time: "2017-02-18T01:01:00Z" }
+  - match: { generated_dest_index.mappings.properties: {} }
+
+  - do:
+      ingest.put_pipeline:
```

Comment:
seems weird to put a pipeline and not use it. I think you might want to set it in the `dest` object for the transform preview.

Reply:
Of course it should be used! Thanks for spotting.